stdlib/2and3/crypt.pyi

```diff
@@ -0,0 +1,18 @@
+import sys
+from typing import List, NamedTuple, Optional
+
+
+if sys.version_info >= (3, 3):
+    class _Method: ...
```

Comment:
This is actually a namedtuple, so we could reflect that here, but this fact is undocumented so I'm ok with leaving it like this.

Reply:
Yeah, I originally implemented it as a NamedTuple, but the documentation suggested that it is meant to be an opaque type, so I reverted to an empty class.